outputs.tf

```diff
@@ -28,24 +28,24 @@ output "public_root_zones" {
 # -------------------------------------------------------------------------------------------------
 # Public Route53 Subdomain Zones (secondary)
 # -------------------------------------------------------------------------------------------------
-#output "local_public_secondary_zones" {
-#  value       = local.public_secondary_zones
+#output "local_public_delegated_secondary_zones" {
```

Comment:
Any reason to keep these lines commented?

Reply:
> Any reason to keep these lines commented?

Yes :-) It's stated at the top of this file. I used the output of locals for debugging to see how they were build. As a real output they are not needed, but still found it useful to keep it in case someone continues to improve this module. I left them commented, as you would have way too much output otherwise (especially stuff that is not required to be used as input for other modules).